pkg/view/component/row_detail.go

```diff
@@ -10,14 +10,23 @@ const (
 )
 
 // AddExpandableDetail allows a row to be expandable
-func (t TableRow) AddExpandableDetail(body Component, replaceContent bool) {
+func (t TableRow) AddExpandableDetail(body ...Component) {
 	er, ok := t[ExpandableRowKey].(*ExpandableRowDetail)
 	if !ok {
-		er = NewExpandableRowDetail(body, replaceContent)
+		er = NewExpandableRowDetail(body...)
 	}
 	t[ExpandableRowKey] = er
 }
 
+func (t TableRow) setReplace(replace bool) {
```

Comment:
A table row shouldn't have this at all. If any of the details set the replace flag to true, it should replace.

Reply:
Without this I can not make the "TestTableRow_AddExpandableDetail" pass

Reply:
Yeah, probably something like:

```
// AddExpandableDetail allows a row to be expandable
func (t TableRow) AddExpandableDetail(details *ExpandableRowDetail) {
	t[ExpandableRowKey] = details
}
```

 and avoid duplication of a second replace

Reply:
Done!